src/consts.js

```diff
@@ -1,35 +1,97 @@
-import openshift from './components/openshift.svg';
-import hcm from './components/hcm.svg';
-import insights from './components/insights.svg';
-import rhelCs from './components/rhel-cs.svg';
+import { CloudCircleIcon, CloudServerIcon, ChartSpikeIcon, OpenshiftIcon } from '@patternfly/react-icons';
 
 export const activeTechnologies = [
     {
+        entitlement: 'insights',
         name: 'insights',
         id: 'insights',
         url: 'insights/actions',
-        icon: insights,
+        icon: ChartSpikeIcon,
+        iconProps: {
+            height: '3.5em',
```

Comment:
why are we passing height and width in these now?

Reply:
Because these icons have different height, they are wider, but PF icons have square size (as they should have), so I had to add these in here to change their size.

Reply:
We shouldn't have to set the height and width, though. You should only have to change one of them because they size relatively.

We were originally using a square layout for the cloud icons because that's what was given to us. However, these icons should not be square because they are supposed to be wider than they are tall. This is why I made https://github.com/RedHatInsights/landing-page-frontend/pull/10

This allows us to change the size of all the icons with one line instead of having to go through this file and change each one individually.

Reply:
I know that this is really bad that we have to change the size of each icon in this file.

Honestly I don't like how font awesome changes width of each icon based on wha is inside of it, if you have icons in front of text this text will jump to right if icon in front of it is wider and you might end up with some strange looking text. That's why PF chose to use square as default dimension (it might have been chosen by mistake, but good mistake).

Reply:
@karelhala [there is a modifier to have them all take up the same width](https://fontawesome.com/how-to-use/on-the-web/styling/fixed-width-icons). However, this should be a modifier and not in the original icon. Icons should not be forced to be a square. However, they should be forced to use the same height.

Reply:
I will probably move these changes to special PR so we are not stuck on this and can move on with entitlements.

Reply:
I just think our best move is to import the openshift and insights icons from PF, and then just consume our own cloud icons. 

Right now, PF is taking the rectangle cloud icons, making them square, then we're making them rectangles again. Let's just use our own, which are rectangles instead of overwriting styles twice.

Either way, can we remove the sizes from the .js file and move then to a .scss file?

Reply:
Yeah, good point! Updated to use our svgs for these two icons.

Reply:
I'll try using PF's icons and passing size to it from css, would that be fine?

Reply:
Not possible sadly, we'll use our svgs and style them properly.